transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/AttachmentHandlerImpl.kt

```diff
@@ -106,6 +108,12 @@ internal class AttachmentHandlerImpl(
         return null
     }
 
+    override fun validate(byteArray: ByteArray): Boolean {
```

Comment:
Since this function is for validating the size of the file attachment, I think it would be clearer to name it `validateSize` or something similar

Reply:
filetype and blocked extension validation will be added to the same function, so the name will remain the same, but validation condition will be extended.